src/main/models/challenged-access/CaseChallengedAccessLogs.ts

```diff
@@ -6,23 +6,21 @@ export interface CaseChallengedAccessLog {
   caseRef: string;
   userId: string;
   action: string;
-  requestStart: string;
   requestEnd: string;
   reason: string;
   timestamp: string;
 }
 
-export const caseChallengedAccessLogOrder = ['caseRef', 'userId', 'requestType', 'action', 'reason', 'requestStartTimestamp', 'requestEndTimestamp', 'timestamp'];
+export const caseChallengedAccessLogOrder = ['userId', 'caseRef', 'requestType', 'action', 'timestamp','reason','requestEndTimestamp'];
 
 export const textMapping: { [key: string]: string } = {
-  caseRef: 'Case Ref',
   userId: 'User ID',
-  requestType: 'Type',
-  action: 'Operation',
-  reason: 'Explanation',
-  requestStartTimestamp: 'Request Start',
-  requestEndTimestamp: 'Request End',
-  timestamp: 'Timestamp (UTC)',
+  caseRef: 'Case ID',
+  requestType: 'Request Type',
+  action: 'Action',
+  timestamp: 'Action On (UTC)',
```

Comment:
Wouldn't it be better to stay consistent and keep timestamp as timestamp? Just saying..